### ISSUE TITLE
Unable to fetch resources

### ISSUE THREAD

**Issue by crawford** [mgattozzi/github-rs#81]
I looks like this was broken in 13cd8755a280196baa991ce8d57e65bd30be8e11.

Given the following code:

```rust
let pr: (_, _, RawPullRequest) = client
        .get()
        .repos()
        .owner(owner)
        .repo(repo)
        .pulls()
        .number(&number.to_string())
        .execute()?;
```

I'm seeing messages like the following in my logs:

```
reqwest::async_impl::response: Response: '404 Not Found' for https://api.github.com/repos/crawford/pulls/2
```

The correct URL is `https://api.github.com/repos/crawford/repo/pulls/2`.

After locally reverting the patch in question, my code works again.

**Comment by mgattozzi** [comment]
Got it. Thanks for the bug report! I'll take a look when I can probably sometime tomorrow.

**Comment by mgattozzi** [comment]
We might want to get this in as part of #79 since it's already fixing the URL and I noticed it wasn't working there.

**Comment by crawford** [comment]
@mgattozzi Since there hasn't been any progress in 10 days, can we revert https://github.com/mgattozzi/github-rs/pull/70? I have some work that is waiting on a resolution.

**Comment by mathstuf** [comment]
Yes, that's fine with me for now. I'll look into it again when I get the chance.

**Comment by mgattozzi** [comment]
@crawford yeah I'll do that now and then merge this in.

**Comment by mgattozzi** [comment]
Reverted #70 in ad65e1d7538bfab116c4702b52c32acc06218ca1
Fixed tests from reversion in 0b57e84f084bc699b64eda5eb962e0f9a56a918f
